packages/react-component-library/src/components/ClassificationBar/ClassificationBar.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+import { ComponentWithClass } from '../../common/ComponentWithClass'
+import { StyledClassificationBar } from './partials/StyledClassificationBar'
+
+export interface ClassificationProps extends ComponentWithClass {
+  /**
+   * Toggles whether the classification is secret or not.
+   */
+  isSecret?: boolean
+  /**
+   * Toggles the styles for when the sidebar is condensed.
+   */
+  inSidebar?: boolean
+  /**
+   * Toggles whether the classification bar is condensed.
+   */
+  isCondensed?: boolean
+}
+
+export const ClassificationBar = ({
+  isSecret,
+  inSidebar,
+  isCondensed,
+  ...rest
+}: ClassificationProps) => {
+  function getClassificationText() {
```

Comment:
We should move this function declaration outside of the body of the component.

That way it doesn't get recreated on ever component re-render.